core/metamodel/src/main/java/org/apache/causeway/core/metamodel/interactions/managed/ManagedProperty.java

```diff
@@ -132,9 +122,8 @@ public Optional<InteractionVeto> modifyProperty(final @Nullable ManagedObject ne
     private ManagedObject reassessPropertyValue() {
         var property = getProperty();
         var owner = getOwner();
-
-        return property.isVisible(owner, InteractionInitiatedBy.FRAMEWORK, getWhere()).isAllowed()
-                && property.isVisible(owner, InteractionInitiatedBy.USER, getWhere()).isAllowed()
+        return property.isVisible(owner, iConstraint().withInitiatedBy(InteractionInitiatedBy.FRAMEWORK)).isAllowed() //TODO strange API
```

Comment:
this is the only place where we ask framework and user visibility;
how is this justified?